packages/venia-ui/lib/RootComponents/Product/product.gql.js

```diff
@@ -1,92 +1,15 @@
 import { gql } from '@apollo/client';
 
+import { ProductDetailsFragment } from './productDetailFragment.gql';
+
 export const GET_PRODUCT_DETAIL_QUERY = gql`
     query getProductDetailForProductPage($urlKey: String!) {
         products(filter: { url_key: { eq: $urlKey } }) {
             items {
-                # Once graphql-ce/1027 is resolved, use a ProductDetails fragment
-                # here instead.
-                __typename
```

Comment:
I believe `__typename` got lost in the copy to the fragment. Can you add it back?